include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
there is some change in latest vs2015. here is link which is included my previous link:
https://msdn.microsoft.com/en-us/library/bb531344.aspx#BK_CRT